src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -291,7 +291,8 @@ def map_to_bools(inputs: Any) -> Any:
 }
 
 
-NUMPY_FUNCTION_TO_SNOWFLAKE_FUNCTION = {
+# Map numpy universal (element-wise) function to Snowflake function
```

Comment:
I would add a comment here that this is specifically for `df.apply(np.<func>)` as opposed to `np.<func>(df)` - we have two maps for ufuncs which are used in different ways - one which allows numpy to map functions to snowpandas, and another which allows snowpandas to map numpy functions to snowflake.

Reply:
Ack, let me add a comment.